docs/learn/data-availability.md

```diff
@@ -0,0 +1,35 @@
+
+## **Stream commits**
+Every stream is an event log consisting of one of more commits, and each commit contains of at least two [IPLD](./glossary.md#ipld) objects. Collectively these commits store the data that makes up the content of a stream.
+
+### Caching
+Ceramic nodes have a built-in caching mechanism for short-term storage of commits. Whenever a Ceramic node performs a [write](../build/writes.md) or a [query](../build/queries.md) on a stream, all commits for that stream are first synced from the [network](./glossary.md#networks) and automatically loaded into that node's in-memory cache. This results in the most popular streams being replicated the most, providing some level of data persistence and availability. However to preserve disk space and node resources, in-memory cache is currently limited to 50 streams. Once that number is reached, the oldest streams will be evicted from the node's cache in order to make room for newer ones. 
+
+If the node happens to shut down or restart, the cache will be cleared. Without sufficient replication across other nodes due to popularity or additional data persistence measures prior to a shutdown, streams that only exist in-memory will be lost forever. Therefore, cache-only is not a dependable source of data availability for a longer period than a specific session.
+
+### Pinning
+Pinning provides a more long-lived mechanism of data persistence for commits. Pinning is a process for instructing a Ceramic node to explicitly host (i.e. "pin") the commits for a specific stream. Since commits are stored in IPLD, Ceramic nodes already contain a bundled [IPFS](./glossary.md#ipfs) node which is where this pinning occurs. IPFS nodes can pin all commits for any stream which is accessible over the Ceramic network to which it is connected. Ceramic pinning can also work using an external IPFS node instead of the bundled internal version.
+
+If developers want the easiest way to make their streams persistent beyond a single session and more resilient against data loss, then pinning is the right option. Ceramic nodes can pin an unlimited number of streams. However, note that if only one IPFS node is pinning a given stream and it disappears forever or gets corrupted, then that stream will be lost. Also, if only one node is pinning a stream (and no other Ceramic nodes have it in cache) and that node goes offline, then that stream will be unavailable to others. Therefore, for improved resilience and data availability it is best to have multiple IPFS nodes running in different envitonments pinning the same streams.
+
+> See the [Pinning](../build/pinning.md) guide for instructions on how to pin streams on a Ceramic node.
+
+### Archiving
```

Comment:
I like the idea of separating the concept of pinning and archiving a bit, but if we do this we should probably do it throughout our stack.  For example, the functionality that enables backing up IPFS commits to filecoin is currently called the FilecoinPinningBackend. Maybe the PinningBackends in js-ceramic should be renamed to ArchivingBackends?

Reply:
sounds good to me